2390 siga wf ajuste destaque aparencia do botao e borda - Altera as cores padrão da borda do destaque, do botão e da borda do botão de azul para amarelo(cor padrão do warning no bootstrap)

cassioaugusto01 opened the pull request:
**Tela Documento Antes da alteração:**
![5 - msg no doc ao clicar em tramitar quando tem um wf vinculado](https://github.com/projeto-siga/siga/assets/37603843/40cc0b80-3138-4354-b62c-15ce5ce10a9c)

**Tela Documento Depois da alteração:**
![7 - bordas amarelas 2](https://github.com/projeto-siga/siga/assets/37603843/2c3b6af4-94fe-4726-bc28-86a6cc6eba6b)

**Alteração para destaque do workflow aplicada somente na tela do documento, a tela do procedimento continua com a mesma aparência:**

![9 - tela procedimento - 24-07-23](https://github.com/projeto-siga/siga/assets/37603843/21fb1dbd-eb9c-4dc5-8433-f822d3c0fcd0)

cassioaugusto01 committed:
Ajuste WF muda a cor padrão do botão do workflow para amarelo com letras pretas.

cassioaugusto01 committed:
Ajuste Siga-WF destaque - altera cor padrão da borda do destaque de azul para amarelo

cassioaugusto01 committed:
Ajuste Siga-WF destaque do documento associado a um WF, alteração na cor padrão da borda dos botões de ação do workflow para amarelo

cassioaugusto01 committed:
Ajuste Siga-WF destaque do documento associado a um WF - adiciona novamente o underline no link do procedimento

cassioaugusto01 committed:
WF ajuste - tela procedimento: botão na cor padrão, tela documento: botão destacado na cor bg-warning do bootstrap

cassioaugusto01 committed:
WF ref: reduz código repetido nos jsps do procedimento e doc diferenciando cor padrão e cor destaque

crivano commented on sigaex/src/main/webapp/WEB-INF/page/exDocumento/exibe.jsp:
```diff
@@ -120,7 +120,11 @@
-	}								
+	}
+	
+	.border-warning {
```
Já é uma classe do Bootstrap veja em: https://getbootstrap.com/docs/4.0/components/card/

crivano commented on sigawf/src/main/webapp/WEB-INF/page/wfApp/doc.jsp:
```diff
@@ -9,10 +9,10 @@
+		<div id="quadro_destaque_tem_workflow_associado" class="card mb-3 border-warning">
 			<div class="card-header bg-warning">
-				<a href="${linkTo[WfAppController].procedimento(pi.id)}"
-					style="color: black; text-decoration: underline;">${pi.sigla}</a> - 
```
Substituir black pela classe text-dark.

cassioaugusto01 committed:
WF ref alteração na Borda, Cor do Texto e Underline pelas classes do Bootstrap

cassioaugusto01 committed:
WF Refatoração: redução de código duplicado no formulário do Workflow em wf/app/procedimento e ex/expediente/doc

cassioaugusto01 commented:
Fiz todas as alterações sugeridas:

- Cor do botão
- Cor do texto do Botão
- Cor da borda do botão
- Uso das classes do Bootstrap
- Redução de repetição de código no formulário

cassioaugusto01 committed:
WF Refatora aparência de destaque para botões de workflow

Em vez de usar uma string com o nome da classe CSS, agora é usado um booleano chamado 'destaque'. Se for true, os botões ficam em destaque(com cores diferentes)
